Description: tests: Adjust stacking tests build check for 2.11 Beta 1 release
 I configured the stacking test binary to only be built when libapparmor
 2.11 is present. The versioning of the 2.11 Beta 1 release (2.10.95)
 causes that check to fail and the stacking tests to not be used.
 .
 This patch adjusts the libapparmor version check to be aware of the 2.11
 Beta 1 versioning.
 .
 Signed-off-by: Tyler Hicks <tyhicks@canonical.com>
 Acked-by: Christian Boltz <apparmor@cboltz.de>
Origin: commit, revision id: tyhicks@canonical.com-20160324165911-dj4sxv0p3q9378ex
Author: Tyler Hicks <tyhicks@canonical.com>
Last-Update: 2016-03-24
X-Bzr-Revision-Id: tyhicks@canonical.com-20160324165911-dj4sxv0p3q9378ex

=== modified file 'tests/regression/apparmor/Makefile'
--- old/tests/regression/apparmor/Makefile	2016-03-19 08:03:18 +0000
+++ new/tests/regression/apparmor/Makefile	2016-03-24 16:59:11 +0000
@@ -167,13 +167,13 @@
     ************************************************************************${nl})
   endif
 
-  ifneq (,$(shell pkg-config --atleast-version 2.11 libapparmor && echo TRUE))
+  ifneq (,$(shell pkg-config --atleast-version 2.10.95 libapparmor && echo TRUE))
     SRC+=stacking.c
     CONDITIONAL_TESTS+=exec_stack stackonexec stackprofile
   else
     $(warning ${nl}\
     ************************************************************************${nl}\
-    Skipping stacking tests: requires libapparmor 2.11 or newer ...${nl}\
+    Skipping stacking tests: requires libapparmor 2.11 Beta 1 or newer ...${nl}\
     ************************************************************************${nl})
   endif
 else

